utilities/transactions/lock/range/range_locking_test.cc

```diff
@@ -0,0 +1,301 @@
+#ifndef ROCKSDB_LITE
+#ifndef OS_WIN
+
+#ifndef __STDC_FORMAT_MACROS
+#define __STDC_FORMAT_MACROS
```

Comment:
what's this used for?

Reply:
This seems to be something no longer needed. I'll remove these.